src/main/resources/com/google/api/codegen/ruby/main.snip

```diff
@@ -175,13 +184,23 @@
       require "{@import.moduleName}"
     @end
 
+    # TODO(landrito): remove these parameters after an appropriate amount of time.
```

Comment:
Nit: since this is autogen code, I think a more generic note is appropriate like,

`Warning: these parameters will be removed on 2017/09/08`.

(I think we can pick a date 8 weeks out or so.)